src/main/java/org/jboss/xavier/analytics/pojo/output/workload/summary/RecommendedTargetsIMSModel.java

```diff
@@ -0,0 +1,121 @@
+package org.jboss.xavier.analytics.pojo.output.workload.summary;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import javax.persistence.*;
+
+@SqlResultSetMapping(
+        name = "mappingRecommendedTargetsIMSModels",
+        classes = @ConstructorResult(
+                targetClass = RecommendedTargetsIMSModel.class,
+                columns = {
+                        @ColumnResult(name = "total", type = Integer.class),
+                        @ColumnResult(name = "rhv", type = Integer.class),
+                        @ColumnResult(name = "osp", type = Integer.class),
+                        @ColumnResult(name = "rhel", type = Integer.class)
+                }
+        )
+)
+
+@NamedNativeQuery(
+        name = "RecommendedTargetsIMSModel.calculateRecommendedTargetsIMS",
+        query = "select count(distinct wi.id) as total, coalesce(sum(case when lower(rt.recommended_targetsims)='rhv' then 1 else 0 end), 0) as rhv, coalesce(sum(case when lower(rt.recommended_targetsims)='osp' then 1 else 0 end), 0) as osp, coalesce(sum(case when lower(rt.recommended_targetsims)='rhel' then 1 else 0 end), 0) as rhel from workload_inventory_report_model_recommended_targetsims rt right join workload_inventory_report_model wi on rt.workload_inventory_report_model_id=wi.id where wi.analysis_id = :analysisId",
```

Comment:
`rhel` has to be changed to `convert2rhel` and i think it's fine to change it just in the query since the UI will show `RHEL` label and not `Convert2RHEL` so i think it's worth only changing the query and nothing else in the code.
```suggestion
        query = "select count(distinct wi.id) as total, coalesce(sum(case when lower(rt.recommended_targetsims)='rhv' then 1 else 0 end), 0) as rhv, coalesce(sum(case when lower(rt.recommended_targetsims)='osp' then 1 else 0 end), 0) as osp, coalesce(sum(case when lower(rt.recommended_targetsims)='convert2rhel' then 1 else 0 end), 0) as rhel from workload_inventory_report_model_recommended_targetsims rt right join workload_inventory_report_model wi on rt.workload_inventory_report_model_id=wi.id where wi.analysis_id = :analysisId",
```